.github/workflows/update-attributions.yml

```diff
@@ -146,8 +146,6 @@ jobs:
           git config --global user.email 'metamaskbot@users.noreply.github.com'
           git commit -am "Update Attributions"
           git push
-        env:
-          GITHUB_TOKEN: ${{ secrets.LAVAMOAT_UPDATE_TOKEN }}
```

Comment:
This token was not used during this step, since `git push` uses the credentials setup during checkout (see that the `git push` step in `update-policies` also doesn't use any token)

Reply:
Thanks for fixing this